src/components/ListItem.css

```diff
@@ -1,14 +1,29 @@
-.ListItem {
-	align-items: baseline;
+/* List item styling */
+.filled-list-view .list-group-item {
+	background-color: #ffe2cd;
+	border-color: #495867;
+	border-width: 1.25px;
+}
+
+.filled-list-view .flex-wrap-item-name {
 	display: flex;
```

Comment:
Thank you both for the heads up, I might take you both up for support on this tomorrow. I did use bootstrap class name styling first but ended up removing and consolidating them into the custom css class names so that the class names were consistent and not overcrowded while allowing me to make the custom changes to the div.